dockerfiles/run/docker-compose.yml

```diff
@@ -10,7 +10,7 @@ services:
       - rabbitmq
       - s3
     entrypoint: bash -c "java -Dworking.directory=/root/ -Dlogback.configurationFile=/root/conf/logback.xml -jar james-server.jar"
-    image: linagora/james-rabbitmq-project:latest
+    image: linagora/james-rabbitmq-project:branch-master
```

Comment:
shouldn't we stick to latest on main docker-compose, which means sticking on last released version?

Reply:
Makes sense.

Reply:
You need to put back the swift setup for this docker-compose then?

And really on docker hub the latest image is 10 months old, our last released version of james is not that old, then something is wrong here?

Reply:
yes, there is something wrong then. Maybe the update has been forgotten during the last release.

We can also provide 2 docker-compose, one for last release and one for dev

we can also use the docker-compose matching the version, so use the tag for the released version, and put your change in the master branch. But then when releasing we have to fix the version in the docker-compose.

Reply:
> You need to put back the swift setup for this docker-compose then?

You are right we need to be consistent.

> And really on docker hub the latest image is 10 months old, our last released version of james is not that old, then something is wrong here?

Likely too.

@naviat maybe?

Also I propose that in the meantime, we have a `docker-compose-branch-master.yml` with S3 setup in the meantime (so both worlds are happy and to ease post release switch)

Reply:
Will take care of it (except for the update of latest image)

Reply:
Done, only the update docker james:latest to the latest stable version of james (3.5) remains